docs/src/man/alignments.md

```diff
@@ -440,3 +440,87 @@ julia> submat['A','B']  # mismatch
 -1
 
 ```
+
+
+## High-throughput sequencing file formats
+
+One of the major sources to generate massive amounts of sequence alignments is
+high-throughput sequencing technologies. The `Bio.Align` module provides several
+data formats commonly used in this field.
```

Comment:
I think this paragraph probably needs revising a bit. The HTS technologies don't generate sequence alignments as this paragraphs leads you to believe, rather sequence alignments are an essential step you do after you have the reads from HTS technologies.

Something like "High-throughput sequencing technologies generate a large amount of data in the form of a large number of nucleotide sequencing reads. One of the most common tasks in bioinformatics is to align these reads against known reference genomes. The `Bio.Align` module provides several data formats commonly used for this kind of task."